front/lib/api/assistant/feedback.ts

```diff
@@ -202,39 +294,46 @@ export async function deleteMessageFeedback(
     throw new Error("Unexpected `auth` without `workspace`.");
   }
 
+  if (!canAccessConversation(auth, conversation)) {
+    throw new Error("User cannot access conversation");
+  }
+
   const message = await Message.findOne({
```

Comment:
No sequelize outside of Resource for new models should live on AgentMessageFeedbackResource